src/app/containers/FrontPageSection/utilities/rowTypes.jsx

```diff
@@ -0,0 +1,36 @@
+import { flatten, isEmpty, isNil } from 'ramda';
+import { TopRow, LeadingRow, RegularRow } from '../../FrontPageStoryRows';
+
```

Comment:
Would be good to add a comment here to add clarity: 
```
// Returns the component that should be used for the Row. This is one of TopRow, LeadingRow.
```
What's the reason that the default returned is not `RegularRow`?

Reply:
Also, renaming it to `rowComponent` would be more accurate. We use type to mean the string 'top' elsewhere, so referring to `rowComponent` here would be good.

Reply:
I've renamed it to `rowComponent`, I've also renamed `TopRow` to `FirstRow` when talking about `TopRow || LeadingRow` to hopefully lessen that confusion.